app/src/main/res/values-bg/strings.xml

```diff
@@ -0,0 +1,746 @@
+<resources>
+    <string name="app_name">"Daily Dozen"</string>
+
+    <string name="beans">"Бобови"</string>
+    <string name="berries">"Горски плодове"</string>
+    <string name="other_fruits">"Други плодове"</string>
+    <string name="cruciferous_vegetables">"Кръстоцветни зеленчуци"</string>
+    <string name="greens">"Зеленолистни зеленчуци"</string>
+    <string name="other_vegetables">"Други зеленчуци"</string>
+    <string name="flaxseeds">"Ленено семе"</string>
+    <string name="nuts">"Ядки"</string>
+    <string name="spices">"Подправки"</string>
+    <string name="whole_grains">"Пълнозърнести"</string>
+    <string name="beverages">"Напитки"</string>
+    <string name="exercise">"Упражнения"</string>
+    <string name="serving_sizes">"Порции"</string>
+    <string name="types">"Видове"</string>
+    <string name="history">"История"</string>
+    <string name="servings_some">"Някои порции"</string>
+    <string name="servings_all">"Всички порции"</string>
+    <string name="servings">"Порция"</string>
+    <string name="about">"Относно"</string>
+    <string name="about_this_app">"Информация за приложението"</string>
+    <string name="latest_videos">"Най-новите видео клипове"</string>
+    <string name="url_latest_videos" translatable="false">"http://nutritionfacts.org/videos"</string>
+    <string name="book_title" translatable="false">"Книгата „Как да не умрем“"</string>
+    <string name="url_book" translatable="false">"http://nutritionfacts.org/book"</string>
+    <string name="donate">"Направете дарение"</string>
+    <string name="url_donate" translatable="false">"https://nutritionfacts.org/donate"</string>
+    <string name="subscribe">"Абонирай се"</string>
+    <string name="url_subscribe" translatable="false">"http://nutritionfacts.org/subscribe"</string>
+    <string name="open_source">"Отворен код"</string>
+    <string name="url_open_source" translatable="false">"http://nutritionfacts.org/open-source"</string>
+    <string name="nutritionfacts_title" translatable="false">"nutritionfacts.org"</string>
+    <string name="url_nutritionfacts" translatable="false">"http://nutritionfacts.org"</string>
+    <string name="name_john_slavick" translatable="false">"Джон Славик"</string>
```

Comment:
Please revert my name back to English. It is interesting to see it translated into Bulgarian though.